_posts/2016-10-07-dream-of-bemifornication.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "Dream of BEMifornication - or how we refactored our scss with linting"
+date: 2016-10-07
+author: axlwaii
+comments: true
+tags:
+  - CSS
+  - SCSS
+  - refactoring
+---
+
+
+Writing a lot of code over the years also means to deal with refactoring at some point. Your skills improve over time and technologies envolve. Coming back to an old project often makes it hard to understand what the heck younger you was thinking when writing and structuring this code. At least that’s the way I sometimes feel. 
```

Comment:
s/envolve/evolve/

s/younger you/Younger You/

I would omit "At least that’s the way I sometimes feel."